usrp_drivers/n200/n200_driver.cpp

```diff
@@ -24,271 +24,339 @@
 #include "utils/driver_options/driveroptions.hpp"
```

Comment:
We would like to see more general overview comments at the beginning of while loops, if statements, etc
<a href="https://imgflip.com/i/21jk8r"><img src="https://i.imgflip.com/21jk8r.jpg" title="made at imgflip.com"/></a>